tests/e2e_logging.py

```diff
@@ -241,15 +244,15 @@ def require_new_response(r):
                 require_new_response(r)
 
             LOG.info("Authenticate as a member, via TLS cert")
-            with primary.client("member0") as c:
+            with primary.client(f"member{member.local_id}") as c:
```

Comment:
Since this construction is so common, should we instead just store this full string as `local_id`? ie - if we never use the `local_id` of `0` in isolation, its only ever to construct `"member0"` or `"user0"`, then we may as well store that string directly (with the bonus that when we've generated a key for `"alice"`, this works, and we don't need to copy her keys to `useralice.pem` and `memberalice.pem`).